tools/ci_build/github/android/build_aar_and_copy_artifacts.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# This script will run build_aar_package.py to build Android AAR and copy all the artifacts
+# to a given folder for publish to Maven Central
+
+set -e
+set -x
+export PATH=/opt/python/cp37-cp37m/bin:$PATH
```

Comment:
this is kind of specific to the docker container.
would it be worth parameterizing some values like path to python, source dir, build dir, and artifacts dir, so this script can be more easily used outside of a CI build?
or it might be worth mentioning that it only is intended to be used with the particular build definition template.

Reply:
This script is particular to the CI build, will add comments,
For build outside of CI pipeline, can directly use build_aar_package.py